TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift

```diff
@@ -40,23 +56,20 @@ class PushFactory {
 }
 
 extension PushFactory: PushFactoryProtocol {
-  func createFactor(withAccessToken accessToken: String, friendlyName: String, pushToken: String, serviceSid: String,
-                    identity: String, success: @escaping FactorSuccessBlock, failure: @escaping TwilioVerifyErrorBlock) {
+  func createFactor(withAccessToken accessToken: String, friendlyName: String, serviceSid: String,
+                    identity: String, pushToken: String?,
+                    success: @escaping FactorSuccessBlock, failure: @escaping TwilioVerifyErrorBlock) {
     do {
       Logger.shared.log(withLevel: .info, message: "Creating push factor \(friendlyName)")
       let alias = generateKeyPairAlias()
       let publicKey = try keyStorage.createKey(withAlias: alias)
       let binding = self.binding(publicKey)
       let config = self.config(withToken: pushToken)
-      let payload = CreateFactorPayload(
-        friendlyName: friendlyName,
-        type: .push,
-        serviceSid: serviceSid,
-        identity: identity,
-        config: config,
-        binding: binding,
-        accessToken: accessToken
-      )
+      let payload = CreateFactorPayload(friendlyName: friendlyName,type: .push,
+                                        serviceSid: serviceSid,identity: identity,
+                                        config: config,binding: binding,
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/7971036277c0571b3d83d953e1f5c0855e226bd3/TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift#L70"></span>There should be no space before and one after any comma.<br />
<code>comma</code> <code>PushFactory.swift:70</code></td>
    </tr>
  </tbody>
</table>